0miner

```diff
@@ -58,441 +61,150 @@ if ps ax | grep SCREEN | grep -v cpuminer | grep -q miner ; then
+  fi
+  $LAUNCH $HCD
+
+## EQUIHASH
+elif [[ $ALGO == EQUIHASH ]];
+then
+  if [[ $EQUIHASH_MINER == DSTM ]]
+  then
+    HCD="${NVOC}/miners/dstm/latest/zm_miner --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --port ${!xport} --pass $MINER_PWD --time $DSTM_OPTS ${!xext}"
+  elif [[ $EQUIHASH_MINER == EWBF ]]
+  then
+    HCD="${NVOC}/miners/ewbf/$EWBF_VERSION/miner --eexit 3 --fee $EWBF_PERCENT --pec --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport} $EWBF_OPTS ${!xext}"
+  elif [[ $EQUIHASH_MINER == BMINER ]]
+  then
+    HCD="${NVOC}/miners/bminer/latest/bminer -uri stratum://${!xaddr}${!xwallet}${!xwork}:$MINER_PWD@${!xpool}:${!xport} $BMINER_OPTS ${!xext}"
+  elif [[ $EQUIHASH_MINER == BMINER_SSL ]]
+  then
+    HCD="${NVOC}/miners/bminer/latest/bminer -uri stratum+ssl://${!xaddr}${!xwallet}${!xwork}:$MINER_PWD@${!xpool}:${!xport} $BMINER_OPTS ${!xext}"
+  fi
+  $LAUNCH $HCD
+
+elif [[ $ALGO == X16R || $ALGO == SIB || $ALGO == LYRA2Z || $ALGO == SKUNK || $ALGO == KECCAK || $ALGO == SKEIN || $ALGO == NEOSCRYPT || $ALGO == LYRA2V2 || $ALGO == TIMETRAVEL10 || $ALGO == PHI || $ALGO == LBRY || $ALGO == DMD-GR || $ALGO == GROESTL || $ALGO == MYR-GR || $ALGO == DECRED || $ALGO == SIA || $ALGO == X13 ]]
+then
+  UCCALGO="-a $(echo $ALGO | sed -e 's/\(.*\)/\L\1/')"
+  HCD="${NVOC}"/miners/"${!xminer}"/ccminer
+  "$LAUNCH" "$HCD" "$UCCALGO" -o stratum+tcp://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" "$CCMINER_OPTS"
+
+## CRYPTONIGHT
+elif [[ $ALGO == CRYPTONIGHT ]]
+then
+  HCD="${NVOC}"/miners/KTccminer-cryptonight/ccminer
+  "$LAUNCH" "$HCD" -o stratum+tcp://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" "$CCMINER_OPTS"
+
+## PASCAL
+elif [[ $ALGO == PASCAL ]]
+then
+  HCD="${NVOC}"/miners/pasc/sgminer
+  UCCALGO="-k pascal"
+  "$LAUNCH" "$HCD" "$UCCALGO" -o stratum+tcp://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" "$CCMINER_OPTS"
+
+  ## CRYPTONIGHTV7
+elif [[ $ALGO == CRYPTONIGHTV7 ]];
+then
+# Remove nvidia.txt if cards count changed
+  if [ -f ~/nvidia.txt ]
+  then
+    if [ "$(grep -c "// gpu:" ~/nvidia.txt)" != "$(nvidia-smi -i 0 --query-gpu=count --format=csv,noheader,nounits)" ]
+    then
+      rm ~/nvidia.txt
+    fi
+  fi
+  # Change verbosity to show hashrate in output log
+  if grep '"verbose_level" : 3' ~/config.txt
+  then
+    sed -i 's/"verbose_level" : 3/"verbose_level" : 4/g' ~/config.txt
+  fi
+  HCD=${NVOC}/miners/xmr-stak/build/bin/xmr-stak_miner
+  screen -c "${NVOC}"/screenrc-miner -dmSL miner "$HCD" --currency monero7 --noCPU --noAMD -o "${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" "$CCMINER_OPTS"
+
+else
+  echo ""
+  echo "COIN/ALGO not found, Check your settings"
+  echo "Miner not started, Stopping watchdog, tempcontrol, ..."
+  pkill -e screen
```

Comment:
@LuKePicci 
Do we have any screens that should stay alive if miner not started?

Reply:
Good point, for sure if tempcontrol has taken control of fans, maybe a good idea to let it run, so it slows down fans as long as gpu are getting cold. However there is probably no reason to kill anything here, just logging the event in the stdout/telegram would be enough.

Reply:
Then lets kill watchdog only